debezium-server/debezium-server-kafka/pom.xml

```diff
@@ -0,0 +1,91 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<groupId>io.debezium</groupId>
+		<artifactId>debezium-server</artifactId>
+		<version>1.5.0-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>debezium-server-kafka</artifactId>
+	<name>Debezium Server Kafka Sink Adapter</name>
+	<packaging>jar</packaging>
+
+	<properties>
+		<skipITs>true</skipITs>
```

Comment:
This should be `false`. The tests should be executed by default as Kafka test can be soelf-contained.